src/views/AddItem.jsx

```diff
@@ -85,7 +86,7 @@ export function AddItem({ listToken, itemList, setData }) {
 								value="14"
 								onChange={handleTime}
 							/>
-							Kind of Soon (14 days)
+							Kind of Soon (7 ~ 30 days)
```

Comment:
I am ok with the days as they currently are. I might add a suggestion though. Doing something like Soon (~7 days) Kind of soon (~30 days) Not that soon (30+ days). It saves character space for one.  And we could then use the same descriptions on the List page and have Inactive show as Inactive (60+ days). Thoughts?

Reply:
My preference would be to keep the days as they currently are, but alter them _ever so slightly_:

Reply:
We could be a little less specific? It's wordier, but makes a little more sense than specific days? 

Soon: within the week
Kind of Soon: within the month
Not That Soon: in more than a month

Soon: this week
Kind of Soon: this month
Not That Soon: next month or beyond

Otherwise, I'd only prefer that a number that ends one timeframe doesn't start another! If I have an item that I want to buy in 7 days, do I put it in Soon or Kind of Soon?! 😆